nova_vm/src/engine/bytecode/bytecode_compiler.rs

```diff
@@ -48,7 +53,26 @@ pub(crate) enum NamedEvaluationParameter {
     ReferenceStack,
 }
 
-pub(crate) struct CompileContext<'agent, 'gc, 'scope> {
+pub(crate) struct JumpTarget {
+    /// Depth of the lexical of the jump target.
```

Comment:
what does a jump target's lexical depth mean?

Reply:
I'll add more comments here, but: it's basically memory of how many declarative environments we had entered when the jump target (loop or switch statement) was created. When a break or continue statement is found, it compares the current number of declarative environments entered (this is tracked in the context) against the jump target memory; this gives us the number of how many environments need to be exited when breaking / continuing.